### ISSUE TITLE
fatal error when no matching pages are found

### ISSUE THREAD

**Issue by splitbrain** [cosmocode/dokuwiki-plugin-structgantt#8]
When the pages for the gantt chart do not exist (eg because they have been deleted) the plugin will throw a fatal error and the page becomes uneditable.

```
PHP Fatal error:  Uncaught Exception: DateTime::__construct(): Failed to parse time string (0) at position 0 (0): Unexpected character in /var/www/farmer-wiki/lib/plugins/structgantt/meta/Gantt.php:349\nStack trace:
#0 /var/www/farmer-wiki/lib/plugins/structgantt/meta/Gantt.php(349): DateTime->__construct('0')\n#1 /var/www/farmer-wiki/lib/plugins/structgantt/meta/Gantt.php(161): dokuwiki\\plugin\\structgantt\\meta\\Gantt->listDays(0, 9223372036854775807, 0, 1)
#2 /var/www/farmer-wiki/lib/plugins/structgantt/meta/Gantt.php(91): dokuwiki\\plugin\\structgantt\\meta\\Gantt->initMinMax()
#3 /var/www/farmer-wiki/lib/plugins/struct/syntax/table.php(101): dokuwiki\\plugin\\structgantt\\meta\\Gantt->__construct(NULL, 'metadata', Object(Doku_Renderer_metadata), Object(dokuwiki\\plugin\\struct\\meta\\SearchConfig))
#4 /var/www/farmer-wiki/inc/parser/renderer.php(111): syntax_plugin_struct_table->render('metadata', Object(Doku_Renderer_metadata), Array)
#5 /var/www/farmer-wiki/inc/parserutils.php(495): Doku_Renderer->plugin('structgantt', Array, 5, '---- s in /var/www/farmer-wiki/lib/plugins/structgantt/meta/Gantt.php on line 349
```

**Comment by mbromley-voxel** [comment]
It seems that this may be linked, but this is the error log that I have been receiving recently. 

`PHP message: PHP Fatal error:  Uncaught Exception: DateTime::__construct(): Failed to parse time string (0) at position 0 (0): Unexpected character in /opt/bitnami/apps/dokuwiki/htdocs/lib/plugins/structgantt/meta/Gantt.php:349\nStack trace:\n#0 /opt/bitnami/apps/dokuwiki/htdocs/lib/plugins/structgantt/meta/Gantt.php(349): DateTime->__construct('0')\n#1 /opt/bitnami/apps/dokuwiki/htdocs/lib/plugins/structgantt/meta/Gantt.php(161): dokuwiki\\plugin\\structgantt\\meta\\Gantt->listDays(0, 9223372036854775807, 0, 1)\n#2 /opt/bitnami/apps/dokuwiki/htdocs/lib/plugins/structgantt/meta/Gantt.php(91): dokuwiki\\plugin\\structgantt\\meta\\Gantt->initMinMax()\n#3 /opt/bitnami/apps/dokuwiki/htdocs/lib/plugins/struct/syntax/table.php(101): dokuwiki\\plugin\\structgantt\\meta\\Gantt->__construct(NULL, 'metadata', Object(Doku_Renderer_metadata), Object(dokuwiki\\plugin\\struct\\meta\\SearchConfig))\n#4 /opt/bitnami/apps/dokuwiki/htdocs/inc/parser/renderer.php(111): syntax_plugin_struct_table->render('metadata', Object(Doku_Renderer_metadata)...'`

**Comment by splitbrain** [comment]
fixed in 039b05b8ed0616ff65edc446d07674c805475bec
